include/counter/model_exec_counter.h

```diff
@@ -0,0 +1,28 @@
+#ifndef MODEL_EXEC_COUNTER_H
+#define MODEL_EXEC_COUNTER_H
+
+#include<iostream>
+#include<map>
+#include<mutex>
+
+using namespace std;
+enum record { RUNTIME_LOAD=1, MODEL_LOAD=2, MODEL_RUN=3, CM_RELEASE=9}; 
+
+/*! \brief class ModelExecCounter
+ */
+class ModelExecCounter
+{
+ public:
+  static void update_model_run_count(std::string model);
+  static void handle_dict(int operation, const std::string model = std::string()); 
+  static std::map<std::string, int> get_dict() { 
+    handle_dict(GETINTERIMDIC); 
+    return inter_dict; 
+  }
+  static std::map<std::string, int > model_dict;
+  static std::map<std::string, int > inter_dict;
+  static const int APPEND = 1;
+  static const int GETINTERIMDIC = 2;
```

Comment:
APPEND and GETINTERIMDIC should be enum

Reply:
Its about constant to a specific class. We can convert them into enum variables.